### PR TITLE
Refactor Cloudflare R2 integration using boto3

### PR DESCRIPTION
**Pull request by pdcp1**
This is part of the issue https://github.com/responsible-ai-collaborative/aiid/issues/2280

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,8 +14,8 @@ RUN apk add --no-cache \
 # install awscli
 RUN pip install awscli
 
-# install wrangler (Cloudflare R2 cli)
-RUN npm install wrangler --global
+# install awscli
```
Comment (kepae) [inline Dockerfile]: boto3*

File: bin/cloudflare_python/cloudflare_delete_object.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import sys
+import boto3
+import cloudflare_client
+
+R2_BUCKET_NAME = sys.argv[4]
```
Comment (kepae) [inline bin/cloudflare_python/cloudflare_delete_object.py]: if we won't use the argparse library and keep it simple, could you please add all script arguments as constants like this, in this file and others? A little more self-documenting and readable.

**Review by kepae — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (pdcp1) [reply Dockerfile]: done

File: bin/cloudflare_python/cloudflare_delete_object.py (hunk 2 above)
Comment (pdcp1) [reply bin/cloudflare_python/cloudflare_delete_object.py]: Agreed. I changed the code to use `argparse`

**Review by kepae — APPROVED** [review]

**Pull request merged**